server/datastore/mysql/migrations/tables/20211202092042_RemovePolicyHistory.go

```diff
@@ -16,6 +16,13 @@ func Up_20211202092042(tx *sql.Tx) error {
 		return errors.Wrap(err, "drop view policy_membership")
 	}
 
+	if _, err := tx.Exec(`ALTER TABLE policy_membership_history
+		DROP FOREIGN KEY fk_policy_membership_policy_id,
+		DROP FOREIGN KEY fk_policy_membership_host_id`,
```

Comment:
Do we understand why we need to drop the index at this point instead of letting them be dropped with the table?

Reply:
When you create the table below (`policy_membership`) it uses the same name for the foreign keys.

Reply:
Ah!